packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -284,13 +276,37 @@ export class SerializedStateManager {
 		}
 	}
 
+	private tryRefreshSnapshot(): void {
+		if (
+			this._refreshSnapshotP === undefined &&
+			this.mc.config.getBoolean("Fluid.Container.enableOfflineSnapshotRefresh") === true
```

Comment:
should this be enabled in the service load tests?

Reply:
you'll probably also want to set the timeout smaller, maybe like 5-10 minutes

Reply:
D:\code\fluid\packages\test\test-service-load, the config is called snapshotRefreshTimeout, i think that would be a better name for the member in this class as well, as its not really expiry

Reply:
I don't know if I can get away with the ILoaderOptionsExperimental approach but let's see. I did it that way because I don't want it to change the ILoaderOptions (I believe it would be a breaking change)